main.qml

```diff
@@ -2034,29 +2033,21 @@ ApplicationWindow {
             appWindow.disconnectedEpoch = Utils.epoch();
         }
 
-        // disconnected longer than 5 seconds?
-        if(disconnected && disconnectedEpoch > 0 && (Utils.epoch() - disconnectedEpoch) >= 5){
-            // for bootstrap mode, first wait until daemon is killed
-            if(appWindow.walletMode === 1 && appWindow.daemonRunning) {
+        if (haveToChangeDaemonNoSyncFlag()) {
+            appWindow.stopDaemon();
+        }
+
+        // disconnected longer than N seconds?
+        if(disconnected && disconnectedEpoch > 0 && (Utils.epoch() - disconnectedEpoch) >= 30){
```

Comment:
It takes 30 seconds now after opening the GUI (simple mode) until the daemon starts. Is this intended?

Reply:
fixed